apps/config/common.py

```diff
@@ -217,7 +218,10 @@ class Common(Configuration):
             "django.db.backends": {"handlers": ["console"], "level": "INFO"},
         },
     }
-
+    LOCAL_DB_PATH = os.path.join(os.path.split(BASE_DIR)[0], "local_db.sqlite3")
```

Comment:
dirname 사용해주시는게 좋을 거 같습니다.
```suggestion
    LOCAL_DB_PATH = os.path.join(os.path.dirname(BASE_DIR), "local_db.sqlite3")
```